worlds/mlss/__init__.py

```diff
@@ -0,0 +1,199 @@
+import typing
+import settings
+from typing import Dict, Any
+from BaseClasses import Tutorial, ItemClassification
+from ..AutoWorld import WebWorld, World
+from .Locations import all_locations, location_table, bowsers, bowsersMini, event, hidden, coins
+from .Options import mlss_options
+from .Regions import create_regions, connect_regions
+from .Rules import set_rules
+from .Items import MLSSItem, itemList, item_frequencies, item_table
+from .Rom import Rom
+from .Names.LocationName import LocationName
+from .Client import MLSSClient
+
+
+class MLSSWebWorld(WebWorld):
+    theme = 'partyTime'
+    bug_report_page = "https://github.com/jamesbrq/ArchipelagoMLSS/issues"
+    tutorials = [
+        Tutorial(
+            tutorial_name='Setup Guide',
+            description='A guide to setting up Mario & Luigi: Superstar Saga for Archipelago.',
+            language='English',
+            file_name='setup_en.md',
+            link='setup/en',
+            authors=['jamesbrq']
+        )
+    ]
+
+
+class MLSSSettings(settings.Group):
+    class RomFile(settings.UserFilePath):
+        """File name of the MLSS US rom"""
+        copy_to = "Mario & Luigi - Superstar Saga (U).gba"
+        description = "MLSS ROM File"
+        md5s = [Rom.hash]
+
+    rom_file: RomFile = RomFile(RomFile.copy_to)
+    rom_start: bool = True
+
+
+class MLSSWorld(World):
+    """
+    Adventure with Mario and Luigi together in the Beanbean Kingdom
+    to stop the evil cackletta and retrieve the Beanstar.
+    """
+    game = "Mario & Luigi Superstar Saga"
+    web = MLSSWebWorld()
+    data_version = 1
+    option_definitions = mlss_options
+    settings = typing.ClassVar[MLSSSettings]
+    item_name_to_id = {name: data.code for name, data in item_table.items()}
+    location_name_to_id = {loc_data.name: loc_data.id for loc_data in all_locations}
+    required_client_version = (0, 4, 3)
```

Comment:
```suggestion
    required_client_version = (0, 4, 4)
```
Unsure if this variable is ever actually used for anything, but 0.4.4 having the Bizhawk Client built in means it makes sense to set it to that.